src/utils/dateUtils.js

```diff
@@ -0,0 +1,11 @@
+import { DISCOUNT_RULES } from "../constants/eventRules.js";
+
+//주말인지 확인하는 함수
+export const isWeekend = (date) => {
+  const day = new Date(2023, 11, date).getDay();
+  return day === 5 || day === 6;
+};
+
+export const isSpecialDay = (date) => {
+  return DISCOUNT_RULES.SPECIAL.DATES.includes(date);
```

Comment:
```suggestion
  const day = new Date(2023, 11, date).getDay();
 return day === 0 || date === 25;
```
이렇게 진행하면 위에 주말인지 구하는것과 동일하게 일요일과 25일 일때 1000원을 할인 할수있을거같습니다!